Userland/Libraries/LibJS/Runtime/Temporal/Duration.cpp

```diff
@@ -1301,46 +1271,40 @@ ThrowCompletionOr<RoundedDuration> round_duration(GlobalObject& global_object, d
         // b. Let dateAdd be ? GetMethod(calendar, "dateAdd").
         auto* date_add = TRY(Value(calendar).get_method(global_object, vm.names.dateAdd));
 
-        // c. Let firstAddOptions be OrdinaryObjectCreate(null).
-        auto* first_add_options = Object::create(global_object, nullptr);
-
-        // d. Let yearsMonthsLater be ? CalendarDateAdd(calendar, relativeTo, yearsMonths, firstAddOptions, dateAdd).
-        auto* years_months_later = TRY(calendar_date_add(global_object, *calendar, relative_to, *years_months, first_add_options, date_add));
+        // d. Let yearsMonthsLater be ? CalendarDateAdd(calendar, relativeTo, yearsMonths, undefined, dateAdd).
+        auto* years_months_later = TRY(calendar_date_add(global_object, *calendar, relative_to, *years_months, nullptr, date_add));
```

Comment:
missing `c.`?

Reply:
good catch!

Reply:
(not missing, just not properly updated after removing one)